BloogBot/AI/SharedStates/EquipBagsState.cs

```diff
@@ -23,7 +23,7 @@ public void Update()
         {
             if (ObjectManager.Player.IsInCombat)
             {
-                botStates.Pop();
+                //botStates.Pop();
```

Comment:
I don't think this change is safe. If the player is in combat, but has an empty bag slot, and a bag in their inventory that needs to be equipped, you'd get stuck in this state forever returning.